libs/ledger-state/src/Cardano/Ledger/State/UTxO.hs

```diff
@@ -236,10 +236,14 @@ countSnapShotStat SnapShot {..} =
 
 data PoolParamsStats = PoolParamsStats
   { ppsPoolId :: !(Stat (KeyHash 'StakePool C))
-  , ppsRewardAcnt :: !(Stat (Credential 'Staking C))
+  , ppsRewardAccount :: !(Stat (Credential 'Staking C))
   , ppsOwners :: !(Stat (KeyHash 'Staking C))
   }
 
+ppsRewardAcnt :: PoolParamsStats -> Stat (Credential 'Staking C)
+ppsRewardAcnt = ppsRewardAccount
+{-# DEPRECATED ppsRewardAcnt "Use `ppsRewardAccount` instead" #-}
+
```

Comment:
This deprecation  isn't needed, we don't release this package. It's more of a benchmarking tool that we used in ledger internally
```suggestion
```

Reply:
In that case, I guess I can just drop the final commit (30032d6de91eda5924565ef7b8f501010a1dc058) as well, right? Since that only renames some internal stuff.

Reply:
Nah, don't drop it. Those renames are useful! Just the deprecation is unnecessary.

Reply:
Ok, thanks for the clarification. It should be all good now (dropped some renames earlier but resurrected them from the reflog without the deprecation helpers).